### PR TITLE
Implement OpenStack Pruner

### PR DESCRIPTION
**Pull request by davelcpanelnet**
Implement OpenStack Pruner

Case RE-745: Implement OpenStack Pruner

Changelog:

### CODE REVIEW

File: .github/workflows/pruner.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: Openstack Pruner
+run-name: Openstack Pruner
+on:
+  schedule:
+    - cron:  '*/5 */1 * * *'
```
Comment (toddr) [inline .github/workflows/pruner.yml:5]: we're going to run this every 5 minutes???

Comment (davelcpanelnet) [reply .github/workflows/pruner.yml:5]: “At every 5th minute past every hour.”

Comment (toddr) [reply .github/workflows/pruner.yml:5]: */5 means every 5 minutes
*/1 means every one hour

**Review by toddr — CHANGES_REQUESTED** [review]
cron settings are not once a day they're currently every 5 minutes.

File: .github/workflows/pruner.yml (hunk 1 above)
Comment (davelcpanelnet) [reply .github/workflows/pruner.yml:5]: Good geez this is why coding on 3 hours of sleep is a bad idea.  fixing

File: .github/workflows/pruner.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: Openstack Pruner
+run-name: Openstack Pruner
+on:
+  schedule:
+    - cron:  '0/5 */1 * * *'
```
Comment (toddr) [inline .github/workflows/pruner.yml:5]: */1 is redundant and confusing. Just do *

Comment (toddr) [reply .github/workflows/pruner.yml:5]: also not 0/5 just 5

`'5 * * * *'`

Comment (davelcpanelnet) [reply .github/workflows/pruner.yml]: OK Handled

**Review by cPholloway — APPROVED** [review]

**Review by toddr — APPROVED** [review]

**Pull request merged**